### ISSUE TITLE
Request for training scripts and reproduction materials

### ISSUE THREAD

**Issue by vegetableno1** [gudo7208/CAD-Coder#1]
Hi Authors,

  Great work on CAD-Coder and congrats on NeurIPS 2025!

  I'm trying to reproduce the results from your paper, but the repository seems to be missing:
  - Training code
  - Data processing scripts
  - Implementation of the geometric reward mentioned in the paper

  Could you please share these materials? This would help the community:
  1. Reproduce your results
  2. Build upon your work
  3. Understand the geometric reward implementation

  Even basic documentation about the training setup would be helpful.

  Thanks!
